cmd/kube-controller-manager/app/plugins.go

```diff
@@ -95,32 +97,18 @@ func ProbeRecyclableVolumePlugins(config componentconfig.VolumeConfiguration) []
 	}
 	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins(nfsConfig)...)
```

Comment:
At this point do we still need to open-code the list or can we iterate an "all plugins" list and find them dynamically?